UI/window-basic-main.cpp

```diff
@@ -9302,6 +9309,16 @@ void OBSBasic::on_stats_triggered()
 	stats = statsDlg;
 }
 
+void OBSBasic::on_menuBasic_MainMenu_Help_aboutToShow()
+{
+#ifdef HAVE_OBSCONFIG_H
+	QString ver;
+	ver += OBS_VERSION;
+	if (!ver.isEmpty())
+		ui->actionReleaseNotes->setVisible(true);
+#endif
```

Comment:
Constructing a string here is probably overkill. As long as the header is present the version should be nonempty.
```suggestion
#ifdef HAVE_OBSCONFIG_H
	ui->actionReleaseNotes->setVisible(true);
#endif
```

Reply:
Good point, fixed.